### PR TITLE
Issue#73 overview graph

### PR DESCRIPTION
**Pull request by mattporritt**

### CODE REVIEW

**Comment by mattporritt** [comment]
@tomdickman 
We'll need to merge in the PR for issue #68 first.

File: lang/en/local_smartmedia.php [hunk 1]
```diff
@@ -45,14 +45,21 @@
...
+$string['dashboard:heading'] = 'Smart Media Dashboard';
 $string['report:duration'] = 'Duration (sec)';
 $string['report:format'] = 'Format';
-$string['report:heading'] = 'Smart Media Report ({$a})';
+$string['report:heading'] = 'Smart Media Overview Report ({$a})';
 $string['report:nocostdata'] = ' - ';
 $string['report:resolution'] = 'Resolution';
 $string['report:size'] = 'Size (Mb)';
 $string['report:summary:filesummary'] = 'All file record summary';
+$string['report:summary:processsummary'] = 'Multimedia file process summary';
```
Comment (tomdickman) [inline lang/en/local_smartmedia.php]: Spelling in 'processsummary'

File: classes/task/report_process.php [hunk 2]
```diff
@@ -127,6 +127,53 @@ private function get_video_file_count() : int {
         return $result;
     }
 
+    /**
+     * Count all the unqiue file objects (contenthashes) for
```
Comment (tomdickman) [inline classes/task/report_process.php]: 'unqiue'

File: classes/output/report_summary.php [hunk 3]
```diff
@@ -167,6 +167,38 @@ private function get_file_summary_totals() : array {
...
+        if (!empty(($values))) { // Handle case where there is no data in table.
+            $uniquemultimediaobjects = $values['uniquemultimediaobjects']->value;
+            $metadataprocessedfiles = $values['metadataprocessedfiles']->value;
+            $transcodedfiles = $values['transcodedfiles']->value;
+
+            $totals = array($uniquemultimediaobjects, $metadataprocessedfiles, $transcodedfiles);
+        }
+
+        return $totals;
+
```
Comment (tomdickman) [inline classes/output/report_summary.php]: Multiple empty lines upsets Travis

File: lang/en/local_smartmedia.php (hunk 1 above)
Comment (mattporritt) [reply lang/en/local_smartmedia.php]: What am I missing?

Comment (tomdickman) [reply lang/en/local_smartmedia.php]: Haha no it's me, you're right, the three 's' put me off, ignore this

**Comment by mattporritt** [comment]
changes made

**Comment by mattporritt** [comment]
@tomdickman This should be has good now. I've rebased this to current dev, so it should merge cleanly with jsut he chagnes relating to this PR

**Pull request merged**